packages/neon-core/__integration__/rpc/clients/NeoServerRpcClient.ts

```diff
@@ -226,6 +226,52 @@ describe("NeoServerRpcClient", () => {
     expect(result).toBe("AOH1BQ==");
   });
 
+  describe.only("findStorage", () => {
+    test("find NEOs totalSupply storage", async () => {
+      const findTotalSupplyPrefixResult = await client.findStorage(
+        contractHash,
+        "0b",
+        0
+      );
+
+      const findTotalSupplyPrefixWithDefaultArgResult =
+        await client.findStorage(contractHash, "0b");
+
+      // This find will only get the totalSupply of NEO. Should be safe and static for usage.
+      expect(findTotalSupplyPrefixResult.truncated).toBe(false);
+      expect(findTotalSupplyPrefixResult.next).toBe(1);
+      expect(findTotalSupplyPrefixResult.results.length).toBe(1);
+      expect(findTotalSupplyPrefixResult.results[0].key).toBe("Cw==");
+      expect(findTotalSupplyPrefixResult.results[1].key).toBe("AOH1BQ==");
+
+      expect(findTotalSupplyPrefixWithDefaultArgResult).toEqual(
+        findTotalSupplyPrefixResult
+      );
+    });
+
+    test("find NEOs balance storage", async () => {
+      // This find will only get a list of accounts on NEOs storage. Will only be used to check the truncated and next values.
+      const findAccountPrefix0Result = await client.findStorage(
+        contractHash,
+        "14",
+        0
+      );
+      expect(findAccountPrefix0Result.truncated).toBe(true);
```

Comment:
unfortunately, within the integration test container, the number of accounts with NEO is <10 so this will naturally fail. I dont think there is another way to do this unless i craft a custom script to spread the neo around 51 addresses.

Reply:
I see, I thought it was synced to the mainnet or testnet. I don't think I'll be able to check a storage where I get more than 50 results. I'll rewrite the test to expect less than 50